### PR TITLE
Improved ansible-galaxy cli STDOUT messages for collections

### PR DESCRIPTION
**Pull request by saito-hideki**
### SUMMARY
Improved ansible-galaxy cli STDOUT messages for collections.

- Fix issue #70010
- Add installation successful message
- This feature targets "collection" sub-command and does not affect "role" sub-command

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
- ansible-galaxy (collections)

##### Before:
```
$ ansible-galaxy collection install netapp.azure                     
Starting galaxy collection install process
Process install dependency map
Starting collection install process
Installing 'netapp.azure:20.6.0' to '/home/foo/work/testing/ansible/collections/ansible_collections/netapp/azure'

$ ansible-galaxy collection download netapp.azure
Starting collection download process to '/home/foo/work/testing/ansible/collections'
Downloading collection 'netapp.azure' to '/home/foo/work/testing/ansible/collections/netapp-azure-20.6.0.tar.gz'
Writing requirements.yml file of downloaded collections to '/home/foo/work/testing/ansible/collections/requirements.yml'
```
##### After:
```
$ ansible-galaxy collection install netapp.azure                
Starting galaxy collection install process
Process install dependency map
Starting collection install process
Installing 'netapp.azure:20.6.0' to '/home/foo/work/testing/ansible/collections/ansible_collections/netapp/azure'
Downloading https://galaxy.ansible.com/download/netapp-azure-20.6.0.tar.gz to /home/foo/.ansible/tmp/ansible-local-41431qt1o1ehp/tmp5ttxfihl
netapp.azure (20.6.0) was installed successfully

$ ansible-galaxy collection download netapp.azure
Starting collection download process to '/home/foo/work/testing/ansible/collections'
Downloading collection 'netapp.azure' to '/home/foo/work/testing/ansible/collections/netapp-azure-20.6.0.tar.gz'
Downloading https://galaxy.ansible.com/download/netapp-azure-20.6.0.tar.gz to /home/foo/.ansible/tmp/ansible-local-83636fd5ug3me/tmpey9gvwgv
Writing requirements.yml file of downloaded collections to '/home/foo/work/testing/ansible/collections/requirements.yml'
netapp.azure (20.6.0) was downloaded successfully
```

### CODE REVIEW

**Comment by saito-hideki** [comment]
recheck

File: lib/ansible/galaxy/collection.py [hunk 1]
```diff
@@ -609,6 +611,7 @@ def download_collections(collections, output_path, apis, validate_certs, no_deps
             display.display("Writing requirements.yml file of downloaded collections to '%s'" % requirements_path)
             with open(to_bytes(requirements_path, errors='surrogate_or_strict'), mode='wb') as req_fd:
                 req_fd.write(to_bytes(yaml.safe_dump({'collections': requirements}), errors='surrogate_or_strict'))
+            display.display("%s (%s) was downloaded successfully" % (name, requirement.latest_version))
```
Comment (s-hertel) [inline lib/ansible/galaxy/collection.py]: If dep_map is empty, `name` is undefined here. This should move into the for loop above so it's displayed for all downloaded.

Comment (saito-hideki) [reply lib/ansible/galaxy/collection.py]: @s-hertel thank you for the review!
I moved this line into the loop and confirmed It working fine as follows:

```
$ ansible-galaxy collection download cisco.nxos cisco.ios
Process install dependency map
Starting collection download process to '/home/foo/work/testing/ansible/collections'
Downloading collection 'cisco.nxos' to '/home/foo/work/testing/ansible/collections/cisco-nxos-0.0.2.tar.gz'
Downloading https://galaxy.ansible.com/download/cisco-nxos-0.0.2.tar.gz to /home/foo/.ansible/tmp/ansible-local-57405pnso4a2i/tmp434o4ow2
cisco.nxos (0.0.2) was downloaded successfully
Downloading collection 'cisco.ios' to '/home/foo/work/testing/ansible/collections/cisco-ios-0.0.2.tar.gz'
Downloading https://galaxy.ansible.com/download/cisco-ios-0.0.2.tar.gz to /home/foo/.ansible/tmp/ansible-local-57405pnso4a2i/tmp434o4ow2
cisco.ios (0.0.2) was downloaded successfully
Downloading collection 'ansible.netcommon' to '/home/foo/work/testing/ansible/collections/ansible-netcommon-0.0.2.tar.gz'
Downloading https://galaxy.ansible.com/download/ansible-netcommon-0.0.2.tar.gz to /home/foo/.ansible/tmp/ansible-local-57405pnso4a2i/tmp434o4ow2
ansible.netcommon (0.0.2) was downloaded successfully
Writing requirements.yml file of downloaded collections to '/home/foo/work/testing/ansible/collections/requirements.yml'
```

Please let us know if you need further correction.

**Review by s-hertel — APPROVED** [review]
Looks good to me. You might want to specify the `collection` subcommand in the changelog since this doesn't affect `ansible-galaxy role`.

**Comment by Akasurde** [comment]
rebuild_merge

**Pull request merged**